packages/backend/src/routers/apiV1Router.ts

```diff
@@ -146,6 +146,21 @@ apiV1Router.get(
         scope: ['profile', 'email'],
     }),
 );
+apiV1Router.get(
+    '/login/gdrive',
```

Comment:
new endpoint for authentication on gdrive with the right scope and refreshtoken ("offline" accesstype)

Reply:
this is awesome!